recipe/meta.yaml

```diff
@@ -36,28 +36,34 @@ about:
     bash, autoconf to compiler autotools on windows. Resulting packages will be MSVC compatible.
     To use this package, in meta.yaml
 
+    ```yaml
+    build:
+      skip: True  # [win and vc<14]
```

Comment:
This gets preprocessed away. Any idea how to prevent that?

Reply:
Oh, that's an annoying thing. Didn't realise that while changing the formatting.

Reply:
Opened an issue https://github.com/conda/conda-build/issues/3751